File: docs/identities/get-started/_common/code-examples/dotnet/setupDev.cs
using System;
using Ory.Client.Api;
using Ory.Client.Client;

public class OryClientFactory
{
    public static FrontendApi CreateOryClient()
    {
        var baseUrl = Environment.GetEnvironmentVariable("ORY_SDK_URL") ?? "http://localhost:4000";

        var config = new Configuration
        {
            BasePath = baseUrl,
        };

        return new FrontendApi(config);
    }
}

File: docs/identities/get-started/_common/code-examples/java/setupDev.java
import sh.ory.ApiClient;
import sh.ory.Configuration;
import sh.ory.api.FrontendApi;

public class OryClientFactory {
    private static final String DEFAULT_BASE_URL = "http://localhost:4000";

    public static FrontendApi createOryClient() {
        String baseUrl = System.getenv().getOrDefault("ORY_SDK_URL", DEFAULT_BASE_URL);

        ApiClient apiClient = Configuration.getDefaultApiClient();
        apiClient.setBasePath(baseUrl);

        return new FrontendApi(apiClient);
    }
}


